### DISCUSSION TITLE
Add Testimonial

### DISCUSSION THREAD

**Discussion by EmanuelGF** [EddieHubCommunity/support#1721] [General] [answered]
Please add my testimonial to the Eddie Hub Testimonials section as suggested on todays livestream 🙏

​> Started contributing and building my GitHub profile and already got a job interview request based on my GitHub activity. So I really appreciate what you do here!

**Comment by eddiejaoude** [comment]
Oh awesome! I missed this, because I was looking for an issue

**Reply by eddiejaoude** [reply]
Thank you 💥

**Comment by SaraJaoude** [comment, marked as answer]
Hi @EmanuelGF I have added this to the website. Thank you for a great testimonial!

**Reply by eddiejaoude** [reply]
Great news! Thank you @SaraJaoude 

@EmanuelGF can you copy and paste it to the discussion where we are collecting all testimonials please https://github.com/EddieHubCommunity/support/discussions/1618

**Reply by EmanuelGF** [reply]
Greetings Sara and Eddie!!! 
Indeed great news ❤🙏

**Reply by EmanuelGF** [reply]
@eddiejaoude I've actually had put it on that discussion on the 24 of May. I initially left it here because I didn´t find the discussion. Sorry my mistake. 😆

**Comment by EmanuelGF** [comment]
I went on to check it out 😎 Seems like the link is not redirecting to my profile.  Can you have look 👍
Looks great, thanks again for including my testimony! 

All the best for you !

**Reply by SaraJaoude** [reply]
Thanks @EmanuelGF this has been fixed
